### PR TITLE
feat: add load and error events to cldimage

### LINKED ISSUE #88: [Feature] Expose `on:load` event on `CldImage`
# **Feature Request**

## **Is your feature request related to a problem? Please describe.**
It's not possible to use the `on:load` event when using the `CldImage` component.

## **Describe the solution you'd like**
Expose the `on:load` event so it can be used.

Example:
```svelte
<CldImage
	src={publicId}
	alt="My image"
	width={256}
	height={256}
	on:load={() => console.log("Image loaded")}
/>
```

## **Describe alternatives you've considered**
For now I've just reverted to using the underlying `@unpic/svelte` `Image` component since it exposes the `on:load` event.

## **Additional context**
It might be as simple as adding `on:load` to the `CldImage` component in [src/lib/components/CldImage.svelte](https://github.com/cloudinary-community/svelte-cloudinary/blob/main/svelte-cloudinary/src/lib/components/CldImage.svelte#L76) like it is done in the `@unpic/svelte` `Image` component: https://github.com/ascorbic/unpic-img/blob/main/packages/svelte/src/lib/image.svelte#L42

### PR DESCRIPTION
**Pull request by codercatdev**
# Description

## Issue Ticket Number

Fixes #88 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Fix or improve the documentation
- [ ] This change requires a documentation update

# Checklist

- [x] I have followed the contributing guidelines of this project as mentioned in [CONTRIBUTING.md](/CONTRIBUTING.md)
- [x] I have created an [issue](https://github.com/cloudinary-community/svelte-cloudinary/issues) ticket for this PR
- [x] I have checked to ensure there aren't other open [Pull Requests](https://github.com/cloudinary-community/svelte-cloudinary/pulls) for the same update/change?
- [x] I have performed a self-review of my own code
- [x] I have run tests locally to ensure they all pass
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes needed to the documentation

### CODE REVIEW

**Comment by colbyfayock** [comment]
hey @codercatdev tested this locally and it works great, thank you!

I'm going to hold off on merging for a minute, we're going to be reverting ENV configuration that was recently released, which should have been on a major release (breaking change) and not sure if we want to do that revert before merging in new stuff

**Review by ghostdevv — APPROVED** [review]

**Pull request merged**
